batch_job/src/amazon_batch_cleanup_script.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+#
+# File: amazon_batch_cleanup_script.sh
+# Author: Kyle Sweeney
+#
+# Created on Jul 17, 2017
+#
+
+configFile=$1
+
+ec2_vpc=$(python -c "import json; print json.load(open('$configFile','r'))['vpc'];")
+ec2_subnet=$(python -c "import json; print json.load(open('$configFile','r'))['subnet'];")
+ec2_security_group_id=$(python -c "import json; print json.load(open('$configFile','r'))['sec_group'];")
+env_name=$(python -c "import json; print json.load(open('$configFile','r'))['env_name'];")
+queue_name=$(python -c "import json; print json.load(open('$configFile','r'))['queue_name'];")
+bucket_name=$(python -c "import json; print json.load(open('$configFile','r'))['bucket'];")
+gateway=$(python -c "import json; print json.load(open('$configFile','r'))['gateway'];")
+
+#disabling jobqueues
+aws batch update-job-queue --job-queue $queue_name --state DISABLED
+
+queue_done_check_output="$(aws batch describe-job-queues --job-queues $queue_name)"
```

Comment:
Suggest that you echo "Deleting queue X" etc as you go along, so the user sees what's happening.